fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/LoanReschedulingWithinCenterTest.java

```diff
@@ -151,7 +151,7 @@ public void testCenterReschedulingLoansWithInterestRecalculationEnabled() {
 
         // VERIFY THE INTEREST
         Float interestDue = (Float) ((HashMap) loanRepaymnetSchedule.get(2)).get("interestDue");
-        assertEquals(String.valueOf(interestDue), "90.82");
+        assertEquals(String.valueOf(interestDue), "42.38");
```

Comment:
As per #682, this, and the other change in line 283 below, doesn't seem right...